stacks/spica/src/storage/components/storage-dialog-overview/storage-dialog-overview.ts

```diff
@@ -12,8 +12,4 @@ export class StorageDialogOverviewDialog {
     public dialogRef: MatDialogRef<StorageDialogOverviewDialog>,
     @Inject(MAT_DIALOG_DATA) public data: any
```

Comment:
```suggestion
    @Inject(MAT_DIALOG_DATA) public data: any
```

You can remove the unused dialogRef property.